tests/kafkatest/services/verifiable_producer.py

```diff
@@ -135,6 +136,9 @@ def _worker(self, idx, node):
                         last_produced_time = t
                         prev_msg = data
 
+                    elif data["name"] == "tool_data":
```

Comment:
This works, but I wonder if it's clearer to have a separate event indicating shutdown so that there is some consistency with VerifiableConsumer?